angular-ngrx-scss/src/app/pull-requests/components/pull-requests-header/pull-requests-header.component.html

```diff
@@ -0,0 +1,24 @@
+<div class="filters-container">
+  <div class="pull-requests-status">
+    <a href="" class="active">
+      <span class="icon" appOcticon="no-entry" size="16"></span>
+      <span>412 Open</span>
```

Comment:
cleanup: Let's make sure the hard coded numbers on lines 5 and 9 are removed before this is merged. :) Should be able to grab those values from the state.